unity/Assets/0Refactored/scenes/experiments/TitrationExperiment/scripts/AcidTitration.cs

```diff
@@ -0,0 +1,300 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using System.IO;
+using System.Linq;
+using System;
+
+public class AcidTitration : MonoBehaviour
+{
+
+	Dictionary<string, int> strongBases = new Dictionary<string, int>() {
+		{"NaOH", 0}, 
+		{"KOH", 1} };
+	Dictionary<string, int> strongAcids = new Dictionary<string, int>() {
+		{"HCl", 0},
+		{"HNO3", 1},
+		{"H2SO4", 2} };
+	Dictionary<string, double> weakAcids = new Dictionary<string, double>() {
+		{"CH3CO2H", 4.75} };
+
+	Dictionary<string, double> weakBases = new Dictionary<string, double>() {
+		{"NH3", 9.25} };
+
+	Dictionary<double, double> result = new Dictionary<double, double>();
+	Dictionary<double, double> equivalenzPoint = new Dictionary<double, double>();
+
+	private double maxRange = 0;
+	private bool checkH2SO4 = false;
+	private bool weakAcid = false;
+	private bool weakBase = false;
+    private bool checkEquivalencePoint = true;
+	private bool baseToggleTitrant = true;
+	private string dropddownAcidText = "HCl";
+	private string dropdownBaseText = "NaOH";
+
+	public string analyteText = "";
+
+	public void calculation(double molTitrant, double mlTitrant, double molAnalyte, double mlAnalyte)

```

Comment:
Please follow the C# naming rule: method names must begin with upper case characters.
But then you have to change everything... Keep this in mind for future changes.